src/views/HomeView.vue

```diff
@@ -164,34 +167,33 @@ const goToBlogDetail = (item) => {
 
 const makeSearch = () => {
   loading.value = true
-  const searchResult = []
   const searchVal = search.value.toLowerCase()
+  searchResultBlog.value = []
```

Comment:
this should be `[]` by default because initial value is [] in the ref
```suggestion
```

Reply:
maybe this should be done in clearSearch() function?

Reply:
ah, so it is to clear the value

Reply:
Yes its for clear every time we make a search request. I do not think we need a separate function to clear since it is only used here!

Reply:
> maybe this should be done in clearSearch() function?

Since we are using this only once, it's fine for now to have it here. If we make use of it multiple times then we can have a separate method for that later.

Reply:
There is already a clearSearch() function in line 200 which is called both by the reset search button in search result (line 104) and the clear search button next to the search text input field (line 23). I think putting this into the appropriate function makes it easier to understand the code...

Reply:
it only clears the search value that we type in the search input. But the array we are mentioning is the metadata of the searched value which we basically get by while searching something.

Reply:
:+1: 
I am fine with either way.

Reply:
Sorry, I was wrong, makeSearch() is the right function to clear the searchResultBlog array.